### PR TITLE
Removes statues from the gold slime pool

### PR DESCRIPTION
**Pull request by jughu**
Yeah those things are great but xenobiology shouldnt get something thats technically invincible health pool wise ((just lots of healtho)

:cl: Improvedname
del: Removes statues from gold slime pool
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Also a buff to xenobio because they're obnoxious

**Comment by Grazyn** [comment]
Learn to use RCDs

**Comment by KieranCat** [comment]
I support this, as I have seen many a round where Xenobio spawns statues with gold slimes and unleashes them upon a hapless station.

**Comment by Armhulen** [comment]
They're not invincible, they take bonus damage from pickaxes allowing them to be killed. They're actually very weak because every single person has lights on them. This change sucks!

**Comment by jughu** [comment]
Xenobiology already has a wide varierity of mobs they can pick from
Changelings, blobbernauts, watchers, goliaths, spiders, skeletons, bears,

they wont be missing a statue mob that is orginally from the wizard idea of turning people into statues then using the staff of animation to animate them temporary.

**Comment by Armhulen** [comment]
I'd rather statues be buffed to viability then removed but i'm also okay with making some mobs rare.

**Comment by ma44** [comment]
Statues can flicker nearby lights as well as having a 10 tile AOE blindness spell.

**Comment by SgtxRootbeer** [comment]
They also are insta-killed my a ripley's drill

**Comment by Bawhoppen** [comment]
Always can wall them in for being eternally immobilized, however they are a bit too memey for my tastes

**Comment by davethwave** [comment]
they can instakill a ling, speaking from experience, if you are a ling and you don't have a light or pda on you, then there's no saving yourself, well except for the last resort which I haven't tested but still but it might kill the headslug too, they are WAY too powerful and the way to get rid of them permanently is not very well known so they normally are just there for the remainder of the round.

**Comment by AnonymousNow** [comment]
Honestly, I just find them to be annoying when they pop up in the gold slime pool. I don't think they're too unbalanced, but they're just enough of a pain in the arse to get rid of.

**Comment by Cyberboss** [comment]
Statues from slimes doesn't fit my headcanon

File: code/modules/mob/living/simple_animal/hostile/statue.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@
 	sight = SEE_SELF|SEE_MOBS|SEE_OBJS|SEE_TURFS
 	anchored = TRUE
 
-	gold_core_spawnable = 1
+	gold_core_spawnable = FALSE
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/statue.dm]: Just remove the line

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Armhulen** [comment]
If we gave them a light that breaks spells we'd be having a different conversation okay but they get a 2 second blind and a flicker lights (doesn't work ever trust me)

**Comment by AnonymousNow** [comment]
Actually does work, just not consistently.

Statues need to be looked at. It seems as though their blind is inconsistent when it comes to mobs, and as player-controlled mobs are likely to be around when player-controlled statues exist, it's a bit of a hassle. They also rarely seem to appear in the mode they're associated with, Wizard - likely because, to be useful, they have to be paired with a staff of animation, which is a staff I've only seen flop wizards use for a minute at a time before they get dunked.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**